Ha ocurrido un error
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
    print(7 / 0)
ZeroDivisionError: division by zero